core/vm/eips.go

```diff
@@ -241,3 +242,32 @@ func enable3860(jt *JumpTable) {
 	jt[CREATE].dynamicGas = gasCreateEip3860
 	jt[CREATE2].dynamicGas = gasCreate2Eip3860
 }
+
+func enable5656(jt *JumpTable) {
+	jt[MCOPY] = &operation{
+		execute:     opMcopy,
+		constantGas: GasFastestStep,
+		dynamicGas:  gasMemoryCopy,
+		minStack:    minStack(3, 0),
+		maxStack:    maxStack(3, 0),
+		memorySize:  memoryMemoryCopy,
+	}
+}
+
+func opMcopy(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]byte, error) {
```

Comment:
Add docstring, with link to EIP spec
```
/// opMcopy implements memcopy as specified in .... 
```

Reply:
addressed in cd09a966defb1ea9ee3d42931f1e9f728fcefaaf